pkg/components/components.go

```diff
@@ -19,6 +19,7 @@ const (
 	SriovDpImageDefault        = "quay.io/booxter/sriov-device-plugin:latest"
 	SriovCniImageDefault       = "docker.io/nfvpe/sriov-cni:latest"
 	KubeMacPoolImageDefault    = "quay.io/kubevirt/kubemacpool:v0.2.0"
+	NMStateHandlerImageDefault = "quay.io/nmstate/kubernetes-nmstate-state-handler:latest"
```

Comment:
Can we tag the nmstate image with a release and change the latest tag here?

Reply:
I will do the release later, we can keep it like this for now. I don't want the PR to be blocked on me